index.js

```diff
@@ -68,18 +100,27 @@ function Car() {
     - Besides the methods on Person.prototype, babies have the ability to `.play()`:
         + Should return a string "Playing with x", x being the favorite toy.
 */
-function Baby() {
-
+function Baby (name, age, favoriteToy) {
+  Person.call(name, age)
+  this.name = name;
+  this.age = age;
+  this.favoriteToy = favoriteToy;
 }
```

Comment:
Because the baby is a person, it will get it's name and age from the person constructor.  You would add it in as an argument, but don't need to assign it again like on lines 105 and 106

Reply:
You are also missing one line of code whose purpose would be to assign the baby the person constructor methods.  Like the ability to eat, poop, and toString name and age.